### DISCUSSION TITLE
Steps are displayed in the "on this page" menu, even though they are not titles or navigation items

### DISCUSSION THREAD

**Discussion by vinnciane** [imfing/hextra#288] [Q&A] [answered]
**Description**

Steps are used mainly in organizing a procedure in documentation.  They should be excluded from the page navigation, as they don't represent any sort of navigation through the page.

**Steps To Reproduce**

I created a procedure with the Steps shortcode.

**Expected Behavior**

The steps should display in body of text only.

**Actual Behavior**

The pretty procedure layout is displayed but also Step 1, step 2, etc. is displayed in the "On this page" navigation.

**Screenshots**

![image](https://github.com/imfing/hextra/assets/45791691/53dfc539-c8c1-4d8d-b846-946b5558b3d1)

![image](https://github.com/imfing/hextra/assets/45791691/8edf7345-340e-4a66-9b02-ac972fffe6ee)

I tried to use bold fencing or an ordered list instead, but I lose the pretty layout intended. 

Thank you.
V.

**Comment by imfing** [comment, marked as answer]
Hi, thanks for this issue. It is by design that the Steps shortcode uses `h3` instead of ordered/unordered lists.

You can override this behavior by:

- create `layouts/shortcodes/steps.html` file to override the default steps shortcode template:

```html
<div class="steps ml-4 mb-12 border-l border-gray-200 pl-6 dark:border-neutral-800 [counter-reset:step]">
    {{ .Inner | markdownify }}
</div>
```

- use the steps shortcode with `<>` angle brackets:

```
{{< steps >}}

### Step 1

This is the first step.

### Step 2

This is the second step.

{{< /steps >}}
```

![image](https://github.com/imfing/hextra/assets/5097752/813d6131-f7b1-41f6-847a-155226656a69)



References:
- https://gohugo.io/content-management/shortcodes/#shortcodes-without-markdown

**Comment by vinnciane** [comment]
Thank you so much !
